packages/material-ui/src/Badge/Badge.test.js

```diff
@@ -170,4 +172,159 @@ describe('<Badge />', () => {
       expect(findBadge(container)).to.have.text('50');
     });
   });
+
+  describe('v5 deprecations', () => {
+    beforeEach(() => {
+      PropTypes.resetWarningCache();
+      stub(console, 'error');
```

Comment:
no `toExpectError` on `master`. Since these warnings are short lived I'm not spending time backporting these matchers.